src/Symfony/Installer/NewCommand.php

```diff
@@ -609,4 +609,17 @@ private function getExecutedCommand()
 
         return sprintf('%s new %s %s', $executedCommand, $this->projectName, $version);
     }
+
+    /**
+     * Checks whether the given directory is empty or not.
+     *
+     * @param  string  $dir the path of the directory to check
+     * @return bool         True if the directory is empty. False otherwise.
```

Comment:
``True if the directory is empty. False otherwise.`` could be removed. The name of the method already makes the meaning of the boolean clear